aggregate/housing_production/hpd_housing_ny_affordable_housing.py

```diff
@@ -91,6 +97,16 @@ def pivot_and_flatten_index(df, geography):
     return df
 
 
+def add_total(df):
+    for t in ["preservation", "newconstruction"]:
+        cols = [
+            f"units_{band}_{t}_count"
+            for band in ["eli", "vli", "li", "mi", "midi", "oi"]
+        ]
+        df[f"units_TOTAL_{t}_count"] = df[cols].sum(axis=1)
```

Comment:
one thing to confirm here is whether the naming is units_TOTAL_{t}_count or in @AmandaDoyle  original issue #291 is _ allami_

Reply:
Oh yea I don't know where I got `TOTAL` from, will re-name